net/paho.mqtt.cpp/Portfile

```diff
@@ -24,9 +24,15 @@ depends_lib-append \
                     port:paho.mqtt.c
 
 # We enable SSL so that the paho.mqtt.cpp library builds without issues
+# TODO: enable static libs as well once this is fixed:
+# https://github.com/eclipse/paho.mqtt.cpp/issues/515
 configure.args-append \
+                    -DPAHO_BUILD_SHARED=ON \
```

Comment:
I assume that building shared libs is the default so adding this line does nothing and so we don't need to increase the revision?

Reply:
I do not increase revision here. `PAHO_BUILD_SHARED` is indeed ON by default, but my concern was that once we enable static libs, everyone seeing `PAHO_BUILD_STATIC` but not `PAHO_BUILD_SHARED` gonna think that we disable dynamic libs for some reason.